starport/pkg/protoanalysis/protoanalysis.go

```diff
@@ -55,7 +59,39 @@ func ParseFile(path string) (File, error) {
 	return files[0], nil
 }
 
-// SearchRecursive recursively finds all proto files under path.
-func SearchRecursive(path string) ([]string, error) {
-	return localfs.Search(path, protoFilePattern)
+// FindMessage finds a message from a
+// specific module declared into the app
+func FindMessage(
```

Comment:
protoanalysis cannot know about sdk, messages, modules nor Go code analysis related primitives (structs).

Reply:
I will refactor the PR